File: rev9.py
import json

def rev():
    with open("fruits.json") as f:
        fruits = json.load(f)
        rev_fruits = {}
        for o_key in fruits:
            o_val = fruits[o_key]
            if o_val not in rev_fruits:
                rev_fruits[o_val] = [o_key]
            else:
                rev_fruits[o_val].append(o_key)
    return rev_fruits

print(rev())

File: rev9.js
const fruits = require('./fruits.json')

const rev_fruits = (og) => {
	rev_fruit = {}
	for (let key in og) {
		let ov = og[key]
		if (ov in rev_fruit == false) {
			rev_fruit[ov] = [key]
		} else {
			rev_fruit[ov].push(key)
		}
	}
	return console.log(rev_fruit)
}

rev_fruits(fruits)
